ta/os_test/os_test.c

```diff
@@ -1068,6 +1068,23 @@ TEE_Result ta_entry_params(uint32_t param_types, TEE_Param params[4])
 	return TEE_SUCCESS;
 }
 
+TEE_Result ta_entry_params_tmpmem(uint32_t param_types, TEE_Param params[4])
```

Comment:
I think a inline comment should explicit what this TA command expects, i.e.:
```c
/*
 * Tests how client can provide null or non-null memref parameters
 * param[0] expected as a 0 byte input mapped memeref.
 * param[1] expected as a 0 byte input not-mapped memeref.
 * param[2] expected as a 0 byte output mapped memeref.
 * param[3] expected as a 0 byte output not-mapped memeref.
```

Reply:
ok